database/README.md

```diff
@@ -252,15 +252,17 @@ Indexes:
 
 Contains all relevant data of fee policies applied to orders during auctions.
 
-Column                    | Type                         | Nullable | Details
---------------------------|------------------------------|----------|--------
- auction_id               | bigint                       | not null | unique identifier for the auction
- order_uid                | bytea                        | not null | 56 bytes identifier linking to the order in the `orders` table
- application_order        | serial                       | not null | the order in which the fee policies are inserted and applied
- kind                     | [PolicyKind](#policykind)    | not null | type of the fee policy, defined in the PolicyKind enum
- surplus_factor           | double precision             |          | percentage of the surplus for fee calculation; value is between 0 and 1
- max_volume_factor        | double precision             |          | cap for the fee as a percentage of the order volume; value is between 0 and 1
- volume_factor            | double precision             |          | fee percentage of the order volume; value is between 0 and 1
+Column                    | Type                      | Nullable | Details
+--------------------------|---------------------------|----------|--------
+ auction_id               | bigint                    | not null | unique identifier for the auction
+ order_uid                | bytea                     | not null | 56 bytes identifier linking to the order in the `orders` table
+ application_order        | serial                    | not null | the order in which the fee policies are inserted and applied
+ kind                     | [PolicyKind](#policykind) | not null | type of the fee policy, defined in the PolicyKind enum
+ surplus_factor           | double precision          |          | percentage of the surplus for fee calculation; value is between 0 and 1
+ max_volume_factor        | double precision          |          | cap for the fee as a percentage of the order volume; value is between 0 and 1
+ volume_factor            | double precision          |          | fee percentage of the order volume; value is between 0 and 1
+ sell_amount              | numeric                   |          | quote's sell amount
+ buy_amount               | numeric                   |          | quote's buy amount
```

Comment:
Don't we already store this information in order_quotes? Do we want to duplicate it?

Reply:
I think these will be different values defining `user` price exchange rate, compared to `order_quotes` where the `amm` price exchange rate is saved.
The `user` price exchange rate takes fee into account.

[Here](https://cowservices.slack.com/archives/C05N1FNP3MX/p1706101158241569?thread_ts=1706098147.756779&cid=C05N1FNP3MX) I wrote about an example.